src/Centreon/Domain/Authentication/UseCase/AuthenticateApi.php

```diff
@@ -77,6 +78,9 @@ public function execute(AuthenticateApiRequest $request, AuthenticateApiResponse
         $this->info(sprintf("[AUTHENTICATE API] Beginning API authentication for contact '%s'", $request->getLogin()));
         $this->deleteExpiredToken();
 
+        /**
+         * @var LocalProviderInterface
+         */
         $localProvider = $this->findLocalProviderOrFail();
```

Comment:
you should not have to do this if return type is declared in `findLocalProviderOrFail`